atom/renderer/api/atom_api_spell_check_client.cc

```diff
@@ -155,62 +148,93 @@ void SpellCheckClient::SpellCheckText(
   base::string16 word;
   int word_start;
   int word_length;
+  std::vector<base::string16> words;
+  auto& word_map = pending_request_param_->wordmap();
```

Comment:
Will this every be populated from a previous run -- do we need to `.clear()` this out?

I think the answer is "no" but am not positive

Reply:
Nope, we don't need to clear it. Ideally, we shouldn't receive another request until the first one is complete, i.e. we call `DidFinishCheckingText` on blink, which happens at the end of this function.